config/service.yml

```diff
@@ -123,7 +123,7 @@ services:
     Invertus\SaferPay\Service\PaymentRestrictionValidation:
         class: Invertus\SaferPay\Service\PaymentRestrictionValidation
         arguments:
-            - !tagged saferpay.paymentrestriction
+            - [ '@Invertus\SaferPay\Service\PaymentRestrictionValidation\ApplePayPaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\BasePaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\KlarnaPaymentRestrictionValidation', '@Invertus\SaferPay\Service\PaymentRestrictionValidation\BasePaymentRestrictionValidation' ]
```

Comment:
show me at least 1 more service that uses [ ] in arguments. Always keep the same logic